### ISSUE TITLE
The arrow keys (history) do not work after exiting out a docker container.

### ISSUE THREAD

**Issue by bplasmeijer** [microsoft/terminal#5467]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.18363.752]
Windows Terminal version (if applicable):  0.10.781.0

Any other software? No
```

# Steps to reproduce

[Video](https://www.dropbox.com/s/hyq1i9am2cboqyr/terminal-arrow-keys.mp4?dl=0)
```
1.) open new PowerShell tab
2.) docker run -it mcr.microsoft.com/powershell:lts-alpine-3.10
3.) type 'exit' inside the docker PowerShell container
4.) it will return to the PowerShell terminal
```
# Expected behavior

I can use my arrow keys again, up/down to get the last commands from history.

# Actual behavior

The arrow keys don't work, and I need to kill the terminal tab.

**Without Windows terminal, the behavior works as expected in a PowerShell window.**

**Comment by DHowett-MSFT** [comment]
Mercifully, this was just fixed with /dup #4873. The shell in the docker container was leaving the arrow keys in a different mode.

**Comment by bplasmeijer** [comment]
> Mercifully, this was just fixed with /dup #4873. The shell in the docker container was leaving the arrow keys in a different mode.

Did test with the new build, works @DHowett-MSFT !
